fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/ambassador/AbstractSumoAmbassador.java

```diff
@@ -1327,6 +1336,39 @@ private void propagateNewRoutes(VehicleUpdates vehicleUpdates, long time) throws
         }
     }
 
+    private void propagateSumoVehiclesToRti(long time) throws InternalFederateException {
+        List<String> routeFileVehicles = getRouteFileVehicles();
```

Comment:
using "getAdded" did not work? Also, please rename the method to `getDepartedVehiclesFromRouteFile` or the like, as the current name suggest that it returns all vehicles from the route file here.

Reply:
got ya